content/components/selectpanel.mdx

```diff
@@ -0,0 +1,222 @@
+---
+title: Select panel
+reactId: selectpanel
+figma:
+description: Select panel is a semantic dialog that allows for complex selection options within an overlay.
+---
+
+import {Box} from '@primer/react'
+import ComponentLayout from '~/src/layouts/component-layout'
+export default ComponentLayout
+
+## Overview
+
+Select panel is used for overlay containers that contain multiple control sets which includes advanced single or multi-selection with a filter. Select panel can be anchored to a trigger, or centered within a view.
+
+Commonly, Select panel is used for metadata application to an Issue or Pull Request, like assignee and label selections, or the branch selector on a repository’s code tab.
+
+## Anatomy
+
+<img
+  width="960"
+  alt="Select panel for assigning labels to an issue. 'Apply labels to this issue' heading, close button in upper right hand corner. Search input with a magnifer icon as a visible label, listbox of label options and the first three are selected with a visual checkbox. The footer contains a button to edit labels, and cancel and apply buttons. Purple labels are overlayed onto Select panel: Overlay (dialog) pointing to the container, Search input pointing to the input, role='listbox' bracket of the listbox content, Optional action control to the button in the lower left corner, Explicit confirm / submission buttons bracket for the apply and cancel buttons in the lower right corner."
+  src="https://user-images.githubusercontent.com/40274682/228329313-7c61fff5-0f2f-4170-b319-723e8a2a3a7d.png"
+/>
+
+## Options
+
+<img
```

Comment:
I noticed the search icon in the `anatomy` image, but it's not included in the first `options` image. Not sure if that was deliberate, but it feels like it should be there!

Reply:
Good call!